elide-core/src/main/java/com/yahoo/elide/core/utils/coerce/converters/ISO8601DateSerde.java

```diff
@@ -15,7 +15,7 @@
 /**
  * Serializes ISO8601 Dates to Strings and vice versa.
  */
-public class ISO8601DateSerde implements Serde<String, Date> {
+public class ISO8601DateSerde implements Serde<Object, Date> {
```

Comment:
Will this require the registration of a new Serde to work around the issue you are seeing or will it work with the default Serde?

Reply:
It works with the default serde from what I tested.

What are your thoughts about modifying the Coerce Util lookup to consider the incoming class type as well cause the reason it was failing earlier was being picked up for <Date, Timestamp> when clearly it was registered for <String, Timestamp> since only the targetType is used for deciding what serde to use.

Reply:
I like the idea of matching both types.  Serdes were originally only intended to convert from primitive types (that are in request/response payloads) to more complex things.   We use CoerceUtil in many other places though now - so it probably ought to consider both types.